casbin_couchbase_adapter/adapter.py

```diff
@@ -17,6 +17,26 @@
 from couchbase.n1ql import STATEMENT_PLUS
 
 
+
+def load_policy_line(tokens, model):
```

Comment:
casbin default Adapter is FileAdapter. It reads line by line code each line is a string. Where as couchbase store data in a list. This above change check if token is a list or string.